app-cpp/app355.cpp

```diff
@@ -140,18 +234,26 @@ std::list<OrbitalData> CalculateOrbitalData(std::vector<sat355::TLE> tleList)
             OrbitalData data(inTLE, out_latdegs, out_londegs, out_altkm);
             orbitalList.push_back(std::move(data));
         }
+    });
+
+    auto& [mutex, outputList] = ioDataList; // C++17 Structured Binding simplifies tuple manipulation
```

Comment:
subjective:
consider moving this line inside the block scope below...
sine the tuple variables& it unpacks are only used inside the block scope